src/UserInput.ts

```diff
@@ -17,10 +17,11 @@ export function askUserBoardPath(): Promise<string> {
   });
 }
 
-export async function askUserMove(): Promise<void> {
-  const userMable = await marble();
-  const userDirection = await direction();
+export async function askUserMove(): Promise<{marblePosition: string, marbleDirection: string}> {
```

Comment:
Just to be sure, how the system works if your user enter a non authorized position ?
I saw above a console log if this user can't move but can he try again another input ?

Reply:
Is it the role to readline ? It seems to be a little magical for me, maybe in this case add a little comment to explain that.

Reply:
Handling errors and asking the user to restart the move will be handled in my next PR.

I am adding a comment for readline, can you tell me if the comment format is ok?